### ISSUE TITLE
Unexpected sceditor-nlf paragrapth with break line after unordered and ordered list

### ISSUE THREAD

**Issue by pnowy** [samclarke/SCEditor#673]
Hi,

we are using sceditor and we noticed unexpected behavior.

When we create simple ordered/unordered list, for example:

- one
- two
- three

and after that list we start new paragraph (by creating next list item and uncheck the list button) the sceditor create unexpected break line within paragraph with weird class. Source HTML:

```
<div>
	<ul>
		<li>
			one 
		</li>
		<li>
			two 
		</li>
		<li>
			three 
		</li>
	</ul><br /> 
</div>
<p class="sceditor-nlf">
	<br />
</p>
```

![change_proportion___dietmap_admin_panel___env__local___ver__1_0](https://user-images.githubusercontent.com/3254609/38382256-a7d9ba2e-3909-11e8-90db-9ebb2b394528.png)

Could you tell what is the purpose of the additional HTML code generated and is there an option to disable this behavior (unfortunately we are working a lot with lists and we would like to remove this additional HTML).

Thanks

**Comment by brunoais** [comment]
That paragraph is there so that the person editing the content can start typing after the list if he chooses to.
It shouldn't appear in the resulting xhtml or BBCode. If it does, then it is a bug.

**Comment by brunoais** [comment]
Which version of SCE are you using?

**Comment by pnowy** [comment]
Version 2.1.0

**Comment by brunoais** [comment]
It's a bug. Either the `xhtml` plugin or SCE's internals will have to be changed accordingly.
Will be handled when able.

**Comment by samclarke** [comment]
Thanks for reporting! This is now fixed in the master branch by PR #675
